system-tests/_scripts/delete-cluster.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+if ! dcos-launch delete; then
```

Comment:
Like before, if you implement what I mentioned above you will now need to specify the cluster info file (`-i $CLUSTER_INFO`)